go.mod

```diff
@@ -52,7 +53,6 @@ require (
 	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
 	kubevirt.io/qe-tools v0.1.6
 	sigs.k8s.io/controller-runtime v0.6.2
-	sigs.k8s.io/controller-tools v0.4.0
```

Comment:
hmmm I wonder why this is gone?  Was it removed in another PR?

Reply:
That is what deps update took out.

Reply:
it was probably used for generating the CDI CRD, which we are not doing now.